### PR TITLE
Add test data for array analysis

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR adds testdata for an array analysis for modules I'm developing. I can't really cut these down- hope they don't prove too large.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -380,6 +380,10 @@ If you cannot find suitable test data on this repository, please contact us on t
     - P53_6samples.cls: a categorical (e.g tumor vs normal) class file format (CLS) example
     - c1.symbols.reduced.gmx: a GMX (Gene MatriX file format) example
 
+  - array_expression:
+    - GSE38751.csv: Sample sheet describing Affy array CEL files
+    - '*.CEL.gz': compressed array CEL files
```
Comment (nvnieuwk) [inline README.md:385]: This file hasn't been added? Also the tarred folder is missing :)

Comment (pinin4fjords) [reply README.md]: Arghh- sorry. I changed my mind how to handle this and forgot to update the docs- thanks for the catch.

Comment (nvnieuwk) [reply README.md]: No problem!!

**Comment by pinin4fjords** [comment]
Thanks @nvnieuwk !

**Pull request merged**